### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by bearcheese**
- [X] Docker Maven plugin
- [x] Update README
- [x] Change Travis.yml to create tag during build

### CODE REVIEW

**Comment by bearcheese** [comment]
@kelemena @bodopeti @andkantor please review.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,25 @@
...
+
+As a result of the new tag a special Travis build will kick in, which generates the release artifacts and the proper Docker image.
+
+### How to set the version explicitly?
+
+```
+./release version [newVersion]
+```
+
+This command will instruct Maven to set a new `SNAPHSHOT` version without doing any release activity. These changes needs to be committed and pushed manually.
```
Comment (kelemena) [inline README.md]: This command will instruct Maven to a release a new version with `version` and then bump versions to `newVersion-SNAPHSHOT`. These changes needs to be committed and pushed manually.

Hm?
Or do I misunderstand something?

File: release.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  MVN_CMD="mvn";
+else
+  MVN_CMD="./mvnw"
+fi
+
+case $1 in
+  cleanup)
+    $MVN_CMD release:clean
+    ;;
+  release)
```
Comment (kelemena) [inline release.sh]: Can we create date variables: today, tomorrow?

File: README.md (hunk 1 above)
Comment (bearcheese) [reply README.md]: No, this command is only to set the version manually, it's like a version bump without having to edit all the related pom.xml files.
This is just a helper, the `release.sh release` does what you mentioned (set release version, push, tag, set new snapshot version, push)

**Pull request merged**